### PR TITLE
How We Work updates

### PR DESCRIPTION
**Pull request by ssatagopan**
# [Sprint Planning update](https://github.com/orgs/OHS-Hosting-Infrastructure/projects/1#card-63184494)

Changes proposed in this pull request:

- Added Conventions section
- Updated Sprint Planning section

### CODE REVIEW

File: docs/how-we-work/README.md [hunk 1]
```diff
@@ -19,6 +19,26 @@ Important points:
...
+   * Documentation: when a ticket is only for documentation
+   * Maintenance: when a ticket is to manage a maintenance window
+   * etc.
+ * Items are estimated using Fibonnaci points of 1, 2, 3, and 5
+   * If any item is estimated as 5, needs to be broken down
+   * 1: Well-understood and can be completed without requiring interaction between teams within 1 day
+   * 2: Well-understood and can be completed with little to no interaction between teams in a few days
+   * 3: Some ambiguity or well understood but requires interaction with other teams that may delay completion
+ * Point estimates are put in parenthesis at end of ticket title
+ * Epics or high level initiative name is put in square brackets at beginning of ticket title
```
Comment (kilbergr) [inline docs/how-we-work/README.md:38]:
```suggestion
  * All team members vote on points, whether or not they're in a technical role. 
  * Epics or high level initiative name is put in square brackets at beginning of ticket title
```

File: docs/how-we-work/README.md [hunk 2]
```diff
@@ -19,6 +19,26 @@ Important points:
...
+  * Bug: defect that needs to be fixed
+  * Task: change that delivers value to client (i.e. application team or end user)
+  * Chore: change that delivers value to team, but not a direct value to client (i.e. lets team work better or easier)
+ * Use additional labels for categorization
+   * Documentation: when a ticket is only for documentation
+   * Maintenance: when a ticket is to manage a maintenance window
+   * etc.
+ * Items are estimated using Fibonnaci points of 1, 2, 3, and 5
+   * If any item is estimated as 5, needs to be broken down
+   * 1: Well-understood and can be completed without requiring interaction between teams within 1 day
```
Comment (kilbergr) [inline docs/how-we-work/README.md:34]:
```suggestion
   * 1: Well-understood and can be completed without requiring interaction between teams in less 1-2 work hours
```

File: docs/how-we-work/README.md [hunk 3]
```diff
@@ -19,6 +19,26 @@ Important points:
...
+   * 2: Well-understood and can be completed with little to no interaction between teams in a few days
```
Comment (kilbergr) [inline docs/how-we-work/README.md:35]:
```suggestion
   * 2: Well-understood and can be completed with little to no interaction between teams within half a day of concentrated work
```

File: docs/how-we-work/README.md [hunk 4]
```diff
@@ -19,6 +19,26 @@ Important points:
...
+   * 3: Some ambiguity or well understood but requires interaction with other teams that may delay completion
```
Comment (kilbergr) [inline docs/how-we-work/README.md:36]:
```suggestion
   * 3: Some ambiguity or well understood but requires interaction with other teams that may delay completion, likely would take 1 full work day
```

**Review by kilbergr — CHANGES_REQUESTED** [review]
So I think your additions are ab fab. However, if this is for the ticket about updating our sprint process, I think there's more to be added. One thing that stands out to me is there's no mention of grooming. Another is that we are doing 1 week sprints.

**Comment by ssatagopan** [comment]
> So I think your additions are ab fab. However, if this is for the ticket about updating our sprint process, I think there's more to be added. One thing that stands out to me is there's no mention of grooming. Another is that we are doing 1 week sprints.

Added Sprint Grooming content, plus adjusted sprint length, start day, and added Definition of Ready as TODO

File: docs/how-we-work/README.md [hunk 5]
```diff
@@ -19,21 +19,55 @@ Important points:
...
+### __Sprint Grooming__
+
+This meeting will occur once a sprint.
+
+We will groom tickets in the backlog to make sure they are ready to review for sprint planning.
+
+Grooming activities include:
+* Adjusting placement of tickets in backlog based on priority
+* Ticket clarification ensuring that Acceptance Criteria is clear
+* Estimation of tickets and splitting of tickets if needed
```
Comment (kilbergr) [inline docs/how-we-work/README.md:69]: What's estimation of tickets? That's not pointing, is it?

File: docs/how-we-work/README.md [hunk 6]
```diff
@@ -19,21 +19,55 @@ Important points:
...
+  * Task: change that delivers value to client (i.e. application team or end user)
+  * Chore: change that delivers value to team, but not a direct value to client (i.e. lets team work better or easier)
+ * Use additional labels for categorization
+   * Documentation: when a ticket is only for documentation
+   * Maintenance: when a ticket is to manage a maintenance window
+   * etc.
+ * Ticket estimation:
+   * Items are estimated using Fibonnaci points of 1, 2, 3, and 5
+     * If any item is estimated as 5, needs to be broken down
+     * 1: Well-understood and can be completed without requiring interaction between teams in less 1-2 work hours
```
Comment (kilbergr) [inline docs/how-we-work/README.md:35]:
```suggestion
     * 1: Well-understood and can be completed without requiring interaction between teams in less than 1-2 work hours
```
Sorry that's my b

**Review by kilbergr — APPROVED** [review]
One missing word and a question, otherwise I think looking good!

File: docs/how-we-work/README.md (hunk 5 above)
Comment (ssatagopan) [reply docs/how-we-work/README.md:69]: Yes, pointing.

Comment (kilbergr) [reply docs/how-we-work/README.md:69]: We don't do pointing in grooming.

Comment (kilbergr) [reply docs/how-we-work/README.md:69]: We COULD split them, though.

Comment (ssatagopan) [reply docs/how-we-work/README.md:69]: hmm...so we only point during sprint planning? it helps provide visibility for the work in backlog if more tickets than just what's for next sprint are estimated.

Comment (kilbergr) [reply docs/how-we-work/README.md:69]: I've always done that way. Happy to try out this suggestion, but then we have the issue of everyone having to be at both grooming and sprint planning. Generally I like to keep grooming smaller. What do you think?

Comment (ssatagopan) [reply docs/how-we-work/README.md:69]: As discussed after standup, it's helpful for full team to be present at both meetings.

**Pull request merged**